Template.yaml

```diff
@@ -0,0 +1,731 @@
+            persistentVolumeClaim:
+              claimName: wso2is-userstore
+          - configMap:
+              defaultMode: 420
+              name: identity-server-conf
+            name: identity-server-conf
+    triggers:
+      - type: ConfigChange 
+      - type: ImageChange
+        imageChangeParams:
+          automatic: true
+          containerNames:
+          - init-is-db
+          from:
+            kind: ImageStreamTag
+            name: wso2is-init-image-stream:latest
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: ${NAME}-rdbms-service-mysql
+  spec:
+    selector:
+      app: wso2is-rdbms-service-mysql
+    type: ClusterIP
+    ports:
+      - name: mysql
+        port: 3306
+        protocol: TCP
+        targetPort: mysql
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: ${NAME}-pattern-ha-service
+  spec:
+    selector:
+      app: wso2is
+      monitoring: jmx  
+    type: ClusterIP
+    ports:
+      - name: servlet-http
+        port: 9763
+        protocol: TCP
+        targetPort: 9763
+      - name: servlet-https
+        port: 9443
+        protocol: TCP
+        targetPort: 9443
+
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: ${NAME}-ha-router
+  spec:
+    port:
+      targetPort: servlet-https
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: passthrough
+    to:
+      kind: Service
+      name: wso2is-pattern-ha-service
+
+parameters:
+- name: OPENSHIFT_PROJECT_NAME
+  displayName: Name of the OpenShift project.
+  required: true
+  value: wso2
+
+- name: NAME
+  displayName: Name of the deployment.
+  required: true
+  value: wso2is
+
+- name: REPLICAS
+  displayName: Number of replicas
+  required: true
+  value: '2'
+
+- name: WSO2_IS_IMAGE_NAME
+  displayName: WSO2 Identity Server image name
+  required: true
+  value: docker.wso2.com/wso2is:5.9.0
+
+- name: IMAGE_PULL_SECRET
+  displayName: WSO2 Subscription Credentials
+  required: true
+  value: wso2-image-pull-secret
+  
+- name: NFS_SERVER_IP
+  displayName: NFS Server IP
+  required: true
+  value: ""
+
+- name: NFS_SHARE_DATABASE
+  displayName: NFS Share Path for database
+  required: true
+  value: ""
+
+- name: NFS_SHARE_USERSTORE
+  displayName: NFS Share Path for userstore
+  required: true
+  value: ""
+
+- name: NFS_SHARE_TENANT
+  displayName: NFS Share Path for tenant
+  required: true
+  value: ""
+
+- name: RESOURCES_LIMITS_CPU
+  displayName: Set a CPU resource limits
+  required: true
+  value: '4000m'
+
+- name: RESOURCES_LIMITS_MEMORY
+  displayName: Set a memory resource limits
+  required: true
+  value: '4Gi'
+
+- name: RESOURCES_REQUEST_CPU
+  displayName: Set a minimum CPU resource limits
+  required: true
+  value: '2000m'
+
+- name: RESOURCES_REQUEST_MEMORY
+  displayName: Set a minimum memory resource limits
+  required: true
+  value: '2Gi'
+
+- name: WSO2_IS_SERVICE_ACCOUNT_NAME
+  displayName:
+  required: true
+  value: wso2svc-account
```

Comment:
Please add a new line.

Reply:
Fix in https://github.com/wso2/openshift-is/pull/1/commits/86e40fc1e706c2890aacee75f5fec80a41ac98a1 commit